pages/common/chsh.md

```diff
@@ -7,9 +7,9 @@
 
 `chsh`
 
-- Change the current user's login shell:
+- Change the login shell of the current user:
 
-`chsh {{path/to/shell}}`
+`chsh -s {{path/to/shell}}`
```

Comment:
I suggest move copy of this page with long option to `linux/` directory. We can use `common/` as smth middle between `osx/` and `linux/`: with long and short options depending on usage frequency. 🤔

Reply:
Could you just merge this, since the page is simply wrong today, and do that later?

Reply:
I'm with Emily on this one.

Reply:
@sourcefrog, please duplicate page with long options to `linux/` and I approve && merge this PR.

Reply:
ok

Reply:
Actually I noticed macOS also does not accept `--shell`, and that's somewhat more common than OpenBSD 😉 . 

I'll use `-s` generically and `--shell` on Linux.